flocker/acceptance/reboot_httpserver.py

```diff
@@ -0,0 +1,44 @@
+"""
+HTTP server used by reboot acceptance tests.
+
+First command-line argument is directory which is expected to be Docker
+volume that will persist across reboots.
+"""
+
+import BaseHTTPServer
+import sys
+import os
+from subprocess import check_output
+
+
+FIRST_BOOT_PATH = os.path.join(sys.argv[1], "first_boot")
+
+
+def boot_time():
+    """
+    :return: ``bytes`` with time this machine was booted.
+    """
+    return check_output([b"uptime", b"--since"])
+
+
+class RebootHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    """
+    Respond to HTTP requests with bootup time of first time process was
+    run, as well as current run's bootup time.
```

Comment:
The boot time doesn't seem relevant.  Any random piece of data would do, so long as it persists.  The data could even be supplied by the test instead of generated by the container - in which case perhaps there is a much simpler implementation of the container.